docs/conf/cincinnati/2018/news/welcome.rst

```diff
@@ -0,0 +1,63 @@
+:template: {{year}}/generic.html
+
+.. post:: Apr 18, 2018
+   :tags: cincinnati-2018, website, cfp, tickets, open-help
+
+Announcing Write the Docs Cincinnati + Open Help Website, Call for Organizers, and CFP
+======================================================================================
+
+Welcome to the official launch of `Write the Docs Cincinnati + Open Help 2018 <http://www.writethedocs.org/conf/cincinnati/2018/>`_! We are excited to announce the first-ever collaboration between Write the Docs and Open Help, coming this August.
```

Comment:
Maybe linkify the WTD and OH first instance (URL for parent community websites)

Reply:
Thank you -- I added the links. looks like the cert for Open Help is still invalid.